packages/shared/forks/ReactFeatureFlags.www-dynamic.js

```diff
@@ -19,7 +19,7 @@ export const disableInputAttributeSyncing = __VARIANT__;
 export const enableFilterEmptyStringAttributesDOM = __VARIANT__;
 export const enableModernEventSystem = __VARIANT__;
 export const enableLegacyFBSupport = __VARIANT__;
-export const enableDebugTracing = __VARIANT__;
+export const enableDebugTracing = !__VARIANT__;
```

Comment:
What's the purpose of this?

Reply:
Oh nvm :)

Reply:
Funny that means the behavior in the old reconciler wasn't being tested :D

Reply:
The new reconciler is enabled when the variant is true. This feature doesn't work so the tests don't pass. This ensures it's enabled in the other variant instead.